pkg/graph/schema.graphql

```diff
@@ -1357,13 +1357,30 @@ type OperationalSolutionSubtask {
   modifiedDts: Time
 }
 
+type ChangedFields {
+  changes: [Field!]!
+}
+
+type Field {
+  name: String!
+  value: ChangedFieldValue!
+}
+
+union ChangedFieldValue = ChangedFields | FieldValue

```

Comment:
@OddTomBrooks, would it make sense for this to be FieldValue and not a ChangdFields? We don't have a nested change hierarchy, so I'm not sure having this nested makes sense.

Reply:
This is something I thought about as well. It was originally designed with support for nested fields in mind. Rather than toss that work out I wanted to maintain it in case we needed it moving forward as it works the same either way. In fact, our processing on the server side does not currently support nested change fields right now - that would be the only change necessary I believe.

Reply:
Thanks for the clarification Tom! I think it's clearer to remove the union type. I think leaving it there suggests that fields could be nested, but we don't have any nested fields, and I don't see how that would change (unless we were to drastically overhaul how we are doing auditing).